modules/gdscript/gdscript.cpp

```diff
@@ -1989,10 +1989,7 @@ String GDScriptLanguage::get_extension() const {
 }
 
 void GDScriptLanguage::finish() {
-	if (_call_stack) {
-		memdelete_arr(_call_stack);
-		_call_stack = nullptr;
-	}
+	_call_stack.free();
```

Comment:
This is only gonna free the stack of the main thread, or I am missing something?

Reply:
This destructor runs on every thread, so it should be fine.